mac/Keyman4MacIM/Keyman4MacIM/ha.lproj/Localizable.strings

```diff
@@ -0,0 +1,62 @@
+/* Message displayed to confirm delete of the Keyman keyboard selected by the user from the keyboard list  */
+"message-confirm-delete-keyboard" = "Ka tabbatar kana son ka goge madanni '%@'?";
+
+/* Delete keyboard confirmation info indicating that the delete action cannot be undone  */
+"info-cannot-undo-delete-keyboard" = "Baza ka fasa wannan ba.";
+
+/* Button text to cancel delete of the specified installed Keyman keyboard  */
+"button-cancel-delete-keyboard" = "Sokewa";
+
+/* Button text to confirm delete of the specified installed Keyman keyboard  */
+"button-delete-keyboard" = "Goge";
+
+/* Message displayed to confirm installation of a keyboard from the specifed .kmp file double-clicked by the user  */
+"message-confirm-install-keyboard" = "A saka madannin Keyman?";
+
+/* Message displayed to confirm installation of a keyboard from the specifed .kmp file double-clicked by the user  */
+"info-install-keyboard-filename" = "Kana son Keyman ya saka madanni daga ma'ajiya '%@'?";
+
+/* Button text to cancel installation of the double-clicked Keyman file  */
+"button-cancel-install-keyboard" = "A saka madannin Keyman";
+
+/* Button text to confirm installation of the double-clicked Keyman file  */
+"button-install-keyboard" = "Shigar";
+
+/* Message displayed to inform user that .kmp file could not be read/unzipped  */
+"message-keyboard-file-unreadable" = "Bazai iya karanta ma'ajiyar Keyman ba '%@'.";
+
+/* Button text to acknowledge that .kmp file could not be read  */
+"button-keyboard-file-unreadable" = "Yayi";
+
+/* label text to identify Keyman version */
+"version-label-text" = "Nau'in: %@";
+
+/* Status text for keyboard downloading  */
+"message-keyboard-downloading" = "Saukewa...";
+
+/* Status text for keyboard download complete  */
+"message-keyboard-download-complete" = "An kammala saukewa.";
+
+/* Button text to cancel downloading keyboard  */
+"button-cancel-downloading" = "Soke";
+
+/* Button text keyboard download complete  */
+"button-download-complete" = "An kammala";
+
+/* keyboards label in the Package Information window */
```

Comment:
Several strings here are not translated to Hausa

Reply:
I pulled in the Hausa strings across all the platforms since it was 99% complete. 

The macOS strings are about 91% complete, so I'm fine with getting the rest of them on future PRs.